nemli/main.py

```diff
@@ -0,0 +1,144 @@
+import discord
+from discord.ext import commands
+import requests
+
+from nemli.config import settings
+
+
+client = commands.Bot(
+    intents=discord.Intents.default(),
+    command_prefix=settings.DISCORD_PREFIX
```

Comment:
```suggestion
    command_prefix=settings.discord.prefix
```

entendeu a ideia? namespaceado e semantico, com nested